README.md

```diff
@@ -26,20 +26,14 @@ an Anaconda installation on Ubuntu, with PyTorch installed.
 
 Install [PyTorch](https://github.com/pytorch/pytorch#installation) if you haven't already.
 
-Install this fork for Warp-CTC bindings:
+Install Warp-CTC bindings:
```

Comment:
Is Warp-CTC still needed? Have you already tried use the native Pytorch CTC? They have fixed several issues regarding the loss (see [#27460](https://github.com/pytorch/pytorch/pull/27460), [#27039](https://github.com/pytorch/pytorch/pull/27039), [#23298](https://github.com/pytorch/pytorch/pull/23298)).  I know that are some issues, but the warp ctc has issues too. I have trained a couple of months ago with the pytorch ctc and I was able to get pretty decent results :+1:

Reply:
Thanks @igormq! I agree, there are still issues with the native CTC loss but it makes support much much easier. I'll do a few test runs locally to see how it stacks up, and if it seems reasonable I'll replace warp-ctc with the native function.

I'm just still hesitant as there are edge cases that are unexplained still in the native CTC function. Atleast warp-ctc I've tried and tested it in many scenarios. 

Can't wait to get rid of warp-ctc 😂